templates/components/datatable.html.twig

```diff
@@ -36,15 +36,29 @@
       {{ __('No data') }}
    </div>
 {% else %}
-    {{ include('components/pager.html.twig', {
-        'count': filtered_number,
-        'additional_params': additional_params ~ '&sort=' ~ sort ~ '&order=' ~ order
-    }) }}
+    {% if nopager is not defined %}
+        {{ include('components/pager.html.twig', {
+            'count': filtered_number,
+            'additional_params': additional_params ~ '&sort=' ~ sort ~ '&order=' ~ order
+        }) }}
+    {% endif %}
 
-    <div class="table-responsive">
+    <div class="table-responsive" {% if showmassiveactions %} id="{{ massiveactionparams['container'] }}" {% endif %}>
+        {% if showmassiveactions %}
+            {% do call('Html::showMassiveActions', [massiveactionparams]) %}
+        {% endif %}
         <table class="table table-hover">
             <thead>
                 <tr>
+                    {% if showmassiveactions %}
+                        <th style="width: 30px;">
+                            <div>
+                                <input class="form-check-input massive_action_checkbox" type="checkbox" id="checkall_{{ massiveactionparams['container'] }}"
+                                    value="" aria-label="{{ __('Check all as') }}"
```

Comment:
I see this label exists for the search results massive action checkboxes, but it doesn't make sense to me.
I should be just "Check all" I think.